src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -215,7 +215,9 @@ private AuthCredentials extractCredentials0(final SecurityRequest request) {
                 }
             }
         }
-        log.error("Failed to parse JWT token using any of the available parsers");
+        if (log.isDebugEnabled()) {
```

Comment:
I'd recommend to just change `log.error()` to `log.debug()` without the `if` statement. The same check as done in the if is just down again inside the call:

https://github.com/apache/logging-log4j2/blob/20035c443e66b9623bde447facde15f54026fa29/log4j-api/src/main/java/org/apache/logging/log4j/spi/AbstractLogger.java#L2312-L2321

As we are just logging a constant without any further stuff, this should not incur any significant overhead.

Additionally, codecov will then see a 100% patch coverage.

Reply:
Removed the check to see if debug is enabled